### DISCUSSION TITLE
系统应用里在android.bp里如何导入

### DISCUSSION THREAD

**Discussion by wanglianhui1-tal** [Tencent/libpag#1751] [Q&A] [answered]
开发者，你好。
目前有个需求想要在系统应用里实现这个方案，在android.bp导入后实现调用load方法错误。
java.lang.UnsatisfiedLinkError: No implementation found for void org.libpag.PAGLayer.nativeInit() (tried Java_org_libpag_PAGLayer_nativeInit and Java_org_libpag_PAGLayer_nativeInit__)

**Comment by domchen** [comment, marked as answer]
请使用标准的aar方式集成，如果自己拆开aar获取里面的so，需要自己加载so。你的报错应该只有两种情况：1.自己拆开了aar，但是没有及时加载so，在没有so的情况下调用了Java API。2.架构配置错误，注意设置一下abifilters相关接口。
